src/Repository/ProductAttributeValueRepository.php

```diff
@@ -31,18 +35,26 @@ public function getUniqueAttributeValues(AttributeInterface $productAttribute, T
         /** @var string|null $storageType */
         $storageType = $productAttribute->getStorageType();
 
-        return $queryBuilder
+        $queryBuilder
```

Comment:
I think it would be good to write integration tests in PHPUnit, as the query seems not to be trivial.

Reply:
Done